benchmark/benchmarks.jl

```diff
@@ -45,37 +43,39 @@ for subdir ∈ subdirs
 
     A = MatrixMarket.mmread(joinpath(iterpath, "K_0.mtx"))
     b = readdlm(joinpath(iterpath, "rhs_0.rhs"), Float64)[:]
-    nnz_A = nnz(tril(A))
+    nnz_A = nnz(tril(A, -1))
+    n = size(A,1)
     if size(A, 1) ≤ 25000
```

Comment:
`n`

Reply:
Should I erase line 47?

Reply:
No. I mean to use `n` instead of `size(A, 1)`.